src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/VanillaItem.java

```diff
@@ -9,5 +9,6 @@
 	
 	public VanillaItem(Category category, ItemStack item, String name, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, name, recipeType, recipe);
+		setVanilla(true);
```

Comment:
Aaaaand doing that pretty much removes all Chain Armor Recipes, you could have just deleted all the code if that is what you wanted to do.